doc/content/getting-started/migrating/migrate-from-chirpstack.md

```diff
@@ -1,43 +1,45 @@
 ---
-title: "Migrating from ChirpStack"
+title: "Migrating End Devices from ChirpStack"
 description: ""
-weight: 4
-aliases: "/getting-started/migrating-from-networks/migrate-from-chirpstack"
+weight: 3
+aliases: "/getting-started/migrating-from-networks/migrate-from-ChirpStack"
 ---
 
-This section contains instructions on how to migrate end devices from ChirpStack to {{% tts %}}.
+This section contains instructions on how to migrate end devices from ChirpStack to {{% tts %}} using [`ttn-lw-migrate` migration tool](https://github.com/TheThingsNetwork/lorawan-stack-migrate).
 
 <!--more-->
 
-## Configure ttn-lw-migrate
+`ttn-lw-migrate` tool is used to export end devices and applications from ChirpStack to a [JSON file]({{< ref "/getting-started/migrating/device-json" >}}). This JSON file can afterwards be [imported in {{% tts %}}]({{< ref "/getting-started/migrating/import-devices" >}}) via [Console]({{< ref "/getting-started/console" >}}) or via [CLI]({{< ref "/getting-started/cli" >}}).
+
+## Prerequisites
 
-End devices and applications can easily be migrated from ChirpStack to {{% tts %}} with the [`ttn-lw-migrate`](https://github.com/TheThingsNetwork/lorawan-stack-migrate) tool. This tool is used for exporting end devices and applications to a [JSON file]({{< ref "getting-started/migrating/device-json.md" >}}) containing their description. This file can later be imported in {{% tts %}} as described in the [Import End Devices in The Things Stack]({{< ref "getting-started/migrating/import-devices.md" >}}) section.
+1. [`ttn-lw-migrate` tool](https://github.com/TheThingsNetwork/lorawan-stack-migrate) version `0.5.0` or higher installed on your system.
```

Comment:
Where is the documentation on how to install this? People are now sent to the repository again where there are no installation instructions. Should they be there or should there be a page here on how to install `ttn-lw-migrate`?

Reply:
I will be working on that in the following days, here's the issue -> https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/316. I didn't want to drag out this PR only because of that.